en/asgardeo/docs/apis/restapis/actions.yaml

```diff
@@ -0,0 +1,859 @@
+            -H 'Content-Type: application/json' \
+            -d '{
+                "name": "Access Token Pre Issue",
+                "description": "This is the configuration of pre-action for issuing access token.",
+                "endpoint": {
+                    "uri": "https://myextension.com/token",
+                    "authentication": {
+                        "properties": {
+                            "username": "auth_username",
+                            "password": "auth_password"
+                        },
+                        "type": "BASIC"
+                    }
+                }
+            }'
+      x-codegen-request-body-name: body
+
+    get:
+      tags:
+        - Actions Endpoint
+      summary: List Actions
+      description: "This API returns actions according to the action type. \n\n
+      - <b>Scope(Permission) required:</b> ``internal_action_mgt_view``\n\n"
+      operationId: getActionsByActionType
+      parameters:
+        - name: actionType
+          in: path
+          description: Name of the Action Type.
+          required: true
+          schema:
+            enum:
+              - preIssueAccessToken
+              - preUpdatePassword
+              - preUpdateProfile
+              - preRegistration
+      responses:
+        '200':
+          description: Actions are found for the given action type.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ActionResponseList'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      x-codeSamples:
+        - lang: Curl
+          source: |
+            curl --location 'https://api.asgardeo.io/t/{organization-name}/api/server/v1/actions/{actionType}' \
+            -H 'Authorization: Bearer {bearer_token}'
+
+  /actions/{actionType}/{actionId}:
+    patch:
+      tags:
+        - Actions Endpoint
+      summary: Update Action
+      description: "This API updates action configurations and return the updated action configurations. \n\n
+      - <b>Scope(Permission) required:</b> ``internal_action_mgt_update``\n\n"
+      operationId: updateAction
+      parameters:
+        - name: actionType
+          in: path
+          description: Name of the Action Type.
+          required: true
+          schema:
+            enum:
+              - preIssueAccessToken
+              - preUpdatePassword
+              - preUpdateProfile
+              - preRegistration
+        - name: actionId
+          in: path
+          description: Id of the Action.
```

Comment:
Unique identifier of the action

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4698/commits/ee4f5ca7c24d3d8dcaca4943b7b5aadb6e16e7fc